scoring_engine/models/team.py

```diff
@@ -84,6 +82,39 @@ def get_round_scores(self, round_num):
     def get_all_blue_teams():
         return session.query(Team).filter(Team.color == 'Blue').all()
 
+    @staticmethod
+    def get_all_blue_teams_cached():
+        teams = []
+        team_ids = Team.get_team_ids()
+        for team_id in team_ids:
+            teams.append(Team.get_team_results_cache(team_id))
+        return teams
+
+    @staticmethod
+    @cache.memoize()
+    def get_team_ids():
+        team_ids = []
+        for team_id, in session.query(Team.id).filter(Team.color == 'Blue'):
+            team_ids.append(team_id)
+        return team_ids
+
+    @staticmethod
+    @cache.memoize()
+    def get_team_results_cache(team_id):
```

Comment:
Hmm, I'm wondering if we should just cache the api calls, and have the engine update it like you have, @RustyBower because eventually, we're gonna be potentially saving large amounts of data into the cache, like very check and all of it's properties. Hmmmmmmm